mandatory/1-fix-functions.js

```diff
@@ -10,20 +10,21 @@
 
 */
 
-function getMood() {
-  let isHappy = true;
+function getMood(bool) {
+  let isHappy = bool;
+
 
   if (isHappy) {
     return "I am happy";
   } else {
     return "I am not happy";
   }
 }
```

Comment:
This works fine, but we can make it even simpler. We can just pass the parameter directly into the if statements (we can call it what we like, here I have called it `isHappy` as before).
```
function getMood(isHappy) {
  if (isHappy) {
    return "I am happy";
  } else {
    return "I am not happy";
  }
}
```